### ISSUE TITLE
Allow Players to Move Around Campaign Map in Multiplayer

### ISSUE THREAD

**Issue by RogueRedHawk** [Anuken/Mindustry-Suggestions#1111]
**Describe the content or mechanics you are proposing.**

All players playing a campaign should be able to play any map on the planet. Every player should be able to click "Resume" from the Planet Map menu (I would recommend changing this to "Go Here"). This should also apply when the Host is launching the Core to a new sector: the core should play an animation and then the players who joined the host will be able to transport to the new sector.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

In 6.0, the campaign is played like a full planet. It's very challenging and makes the user manage several things at once, including defending captured sectors. That being said, it doesn't really make sense that the players can only be in one sector at a time, when (from a UX standpoint) all the sectors are actively running.

Playing multiplayer co-op on the campaign is a ton of fun, makes you feel strategic, and is overall a great time, but it can get hectic. It doesn't feel very good as a player to have sectors recaptured just because you couldn't quickly return to them without kicking everyone out of the game, especially when you have 4 friends, one of whom could have gone back to the sector and made sure it was performing. This mechanic would fix that frustration by allowing players to feel less constrained by the requirement that all players must be in the same sector.

Additionally, allowing this mechanic would be awesome because it could allow a sector to become a central base. The host could launch the core and a player could stay back and launch resources to the new sector. When on an attack map, if you run out of something a player could quickly move to another sector and reroute items to launch to the sector in peril.

I honestly don't know enough to determine whether this mechanic is feasible from a hardware/software standpoint. In my (non-coding experienced) mind, I would think that the host would not necessarily have to run all the maps in parallel, and perhaps the joined players could run the maps they transport to on their local devices.

In conclusion, this single change would be (in my opinion) a fantastic feature for the 6.0 campaign.

Before making this issue, place an X in the boxes below to confirm that you have acknowledged them. Failure to do so may result in your request being closed automatically.

[x] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
[x] I have checked the Trello to make sure my suggestion isn't planned or implemented in a development version.
[x] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.

**Comment by QmelZ** [comment]
> [x] I have checked the Trello to make sure my suggestion isn't planned or implemented in a development version.

please read what you agree with

**Comment by RogueRedHawk** [comment]
>please read what you agree with

Where on the Trello is this suggestion? I read the Trello at least once per week and I've never seen this specific suggestion.

**Comment by 50275** [comment]
You can run individual maps and host them on a server, but hosting 100+ maps at the same time... good luck. Also this has been suggested countless times before

**Comment by RogueRedHawk** [comment]
@50275 

>but hosting 100+ maps at the same time

I'm not suggesting that. Only the maps with players on them should be actively open. Sectors without players should have their normal passive calculations. If it's going to be a strain, the host could hand off hosting to one of the other players, I'm not sure if that's how it would work on a technical level.

**Comment by RogueRedHawk** [comment]
@50275 

Also where has this been suggested? I looked for a quite awhile and frequent this repo and I haven't seen any posts about it.

**Comment by RogueRedHawk** [comment]
Additionally, it seems unfair that enemies can be attacking you in two sectors whole you and your friends can only be in 1.

**Comment by RogueRedHawk** [comment]
@QmelZ @50275
If you're going to claim that this suggestion existed before I posted it at least prove it.

**Comment by RogueRedHawk** [comment]
Duplicate of #459 and #916

**Comment by godotgamedeveloper** [comment]
Duplicate of this one also: https://github.com/Anuken/Mindustry-Suggestions/issues/1046.
